lib/bundler/audit/cli.rb

```diff
@@ -60,10 +61,10 @@ def check
       end
 
       desc 'update', 'Updates the ruby-advisory-db'
-      def update
+      def update(quiet = false)
```

Comment:
We may need to add other options in the future. I like to avoid changing argument types, as that technically changes the API.